src/coreclr/src/inc/corinfo.h

```diff
@@ -742,7 +742,8 @@ enum CorInfoUnmanagedCallConv
     CORINFO_UNMANAGED_CALLCONV_C,
     CORINFO_UNMANAGED_CALLCONV_STDCALL,
     CORINFO_UNMANAGED_CALLCONV_THISCALL,
-    CORINFO_UNMANAGED_CALLCONV_FASTCALL
+    CORINFO_UNMANAGED_CALLCONV_FASTCALL,
+    CORINFO_UNMANAGED_CALLCONV_MANAGED
```

Comment:
@sandreenko would you prefer if I made a new enum to use instead of extending the CorInfoUnmanagedCallingConvention enum for this use case?

Reply:
The comment just above this says
```
	    // These correspond to CorUnmanagedCallingConvention
```
which is no longer true. Perhaps update that comment too? And I think it's also worth explaining what the newly added convention means, as the name sounds contradictory.

Reply:
I've added some comments to explain the new extended usage of this enum. Let me know what you think!

Reply:
Can `CORINFO_UNMANAGED_CALLCONV_MANAGED` be defined as `CORINFO_UNMANAGED_CALLCONV_UNKNOWN`? It would make it possible to have the definition to be local to the JIT.

Reply:
I've gone and done that.